features/admin.applications.v1/components/share-application-status-response-list.scss

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+ .share-application-status-response-list-header {
```

Comment:
Let's wrap these styles with the class selector of the wrapper component of the share-application-status-response-list component so that these styles are scoped to only that component and the rest of the styles will not be affected.

Reply:
This seems not addressed.

Just to be clear, we need to wrap the styles with the class selector of parent wrapper component, instead of appending the wrapper component class to every class name.

eg:

```scss
.share-application-status-response-list {
   .org-header {
   
   }
   
   .org-id {
   
   }

   .org-id-wrapper {
   
   }
}
```